lib/push.js

```diff
@@ -255,10 +269,29 @@ module.exports = function (log, db, config) {
     },
 
     /**
+     * Notifies a set of devices that the account no longer exists
+     *
+     * @param {Buffer} uid
+     * @param {Device[]} devices
+     * @promise
+     */
+    notifyAccountDestroyed: function notifyAccountDestroyed(uid, devices) {
+      var data = Buffer.from(JSON.stringify({
+        version: PUSH_PAYLOAD_SCHEMA_VERSION,
+        command: PUSH_COMMANDS.ACCOUNT_DESTROYED,
+        data: {
+          uid: uid.toString('hex')
```

Comment:
Including the uid seems like the Right Thing here so that the receiver can sanity-check stuff, but I see that none of the other push messages appear to include it.  Should they, for consistency?